test/e2e_node/services/services.go

```diff
@@ -239,7 +244,7 @@ func (es *e2eService) getLogFiles() {
 				continue
 			}
 			glog.Infof("Get log file %q with journalctl command %v.", targetFileName, logFileData.journalctlCommand)
-			out, err := exec.Command("sudo", append([]string{"journalctl"}, logFileData.journalctlCommand...)...).CombinedOutput()
+			out, err := exec.Command("journalctl", logFileData.journalctlCommand...).CombinedOutput()
```

Comment:
We do need the `sudo` right? Or is this process being run as `root` under all settings?